doc/providers/totp.rst

```diff
@@ -57,43 +57,80 @@ and the period of the temporary codes.
     Be warned, custom configurations will not be compatible with the defaults of Google Authenticator app any more. You
     will have to use another application (e.g. FreeOTP on Android).
 
-.. code-block:: php
-
-   <?php
+.. configuration-block::
+    .. code-block:: php
```

Comment:
Here, let's add a blank line between `configuration-block` and `clode-block` directives.

Reply:
Done! Thanks for the feedback